sections/expansion_content.tex

```diff
@@ -510,41 +517,52 @@ \subsection*{You haven't had enough yet?}
 Take a look at the \textbf{fanmade expansions and contents} that the \textbf{HoMM3 BG Community} has developed:
 
   \begin{itemize}
-    \begin{minipage}{5.5cm}
+    \iftoggle{printable}{
+    \begin{minipage}{5.5cm}}
     \item The \href{https://github.com/qwrtln/Homm3BG-mission-book}{Fanmade Mission Book} contains many well playtested Scenarios with unique Scenario rules for even more spice in your games.
+    \iftoggle{printable}{
     \end{minipage}
     \hfill
     \begin{minipage}{2.5cm}
         \includegraphics[width=\linewidth]{\qr/mission-book.png}
     \end{minipage}
-    \begin{minipage}{5.5cm}
+    }
+    \iftoggle{printable}{
+    \begin{minipage}{5.5cm}}
     \item The \href{https://github.com/piotrbruzda/Homm3BG-FactoryRulebook}{Fanmade Factory Expansion} introduces the beloved missing Faction.
     Print it yourself and learn a new Faction to master.
+    \iftoggle{printable}{
     \end{minipage}
     \hfill
     \begin{minipage}{2.5cm}
         \includegraphics[width=\linewidth]{\qr/factory.png}
     \end{minipage}
+    }
   \end{itemize}
   \begin{multicols}{2}
       \includegraphics[width=1.2\linewidth]{\images/mission-book-title.png}
       \includegraphics[width=1.2\linewidth]{\images/factory-book-title.png}
   \end{multicols}
   \begin{itemize}
-    \begin{minipage}{5.5cm}
+    \iftoggle{printable}{
+    \begin{minipage}{5.5cm}}
     \item Bring your own creative ideas to live by building own Scenarios with the help of the \href{https://zedero.github.io/homm3boardgame/}{Scenario Map Editor}.
+    \iftoggle{printable}{
     \end{minipage}
     \hfill
     \begin{minipage}{2.5cm}
         \includegraphics[width=\linewidth]{\qr/map-editor.png}
     \end{minipage}
     \begin{minipage}{5.5cm}
+    }
     \item Generate own custom Heroes and their unique Specialty with the  \href{https://k-adam.github.io/Homm3_hero_creator/}{Hero Creator} and let these Heroes write history.
+    \iftoggle{printable}{
     \end{minipage}
```

Comment:
you can't interleave different kind of blocks: if should be fully within minipage, or vice versa